css/components/nav.css

```diff
@@ -0,0 +1,24 @@
+.nav__list {
+  list-style-type: none;
```

Comment:
list-style: none
would be enough, but it's not wrong

Reply:
cool, sortcut is helpful